load.go

```diff
@@ -110,35 +110,36 @@ func (ld Loader) Load(cfg interface{}) (cmd string, args []string, err error) {
 		panic(fmt.Sprintf("cannot load configuration into %T", cfg))
 	}
 
+	if args, err = ld.load(v); err != nil {
+		return
+	}
+
 	if len(ld.Commands) != 0 {
-		if len(ld.Args) == 0 {
+		if len(args) == 0 {
 			err = errors.New("missing command")
 			return
 		}
 
 		found := false
 		for _, c := range ld.Commands {
-			if c.Name == ld.Args[0] {
-				found, cmd, ld.Args = true, ld.Args[0], ld.Args[1:]
+			if c.Name == args[0] {
+				found, cmd = true, args[0]
+				args = args[1:]
```

Comment:
Weird style here, either put all assignments on the same line or all on different lines:
```go
found, cmd, args = true, args[0], args[1:]
```
```go
found = true
cmd = args[0]
args = args[1:]
```